frontend/src/components/activePage/TableInstance/TableInstanceLogic.tsx

```diff
@@ -71,32 +72,40 @@ const TableInstanceLogic: FC<ITableInstanceLogicProps> = ({ ...props }) => {
           if (!data?.updateInstance?.instance) return prev;
 
           const { instance, updateType } = data?.updateInstance;
-          let isPrettyNameUpdate = false;
+          let notify = false;
 
           if (prev.instanceList?.instances) {
             let instances = [...prev.instanceList.instances];
-            if (updateType === UpdateType.Deleted) {
-              instances = instances.filter(matchK8sObject(instance, true));
-            } else {
-              const found = instances.find(matchK8sObject(instance, false));
-              if (found) {
-                isPrettyNameUpdate = !comparePrettyName(found, instance);
-                instances = instances.map(replaceK8sObject(instance));
-              } else {
+            const found = instances.find(matchK8sObject(instance, false));
+            const objType = getSubObjType(found!, instance, updateType!);
```

Comment:
```suggestion
            const found = instances.find(matchK8sObject(instance, false));
            if (!found) {
              console.error('Invalid update received');
              return;
            }
            const objType = getSubObjType(found!, instance, updateType!);
```

Reply:
i need to send also an undefined value for "found" variable, because the `getSubObjType` internally checks this undefined to know that is an addition

Reply:
In this case then the `!` should not be needed at all